test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -54,30 +53,34 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     const expectSearchCount = async (type: 'ese' | 'esql', searchCount: number) => {
-      await retry.try(async () => {
-        if (searchCount === 0) {
-          await browser.execute(async () => {
-            performance.clearResourceTimings();
-          });
-        }
-        await waitForLoadingToFinish();
-        const endpoint = type === 'esql' ? `${type}_async` : type;
-        const requests = await browser.execute(() =>
-          performance
-            .getEntries()
-            .filter((entry: any) => ['fetch', 'xmlhttprequest'].includes(entry.initiatorType))
-        );
+      await retry.tryWithRetries(
```

Comment:
I've rewrote this part, because it's simpler for debugging. when starting this test, going to get a coffee, `tryWithRetries` failes earlier than `try`. And it makes no sense here to wait too long